src/App.tsx

```diff
@@ -30,6 +31,7 @@ function App() {
               <Route path="/about" element={<AboutMe />} />
               <Route path="/contact" element={<ContactPage />} />
               <Route path="/product-series" element={<ProductSeriesPage />} />
+              <Route path="/products-page" element={<ProductsPage />} />
```

Comment:
Check links in navbar and footer if they guide to correct links

Reply:
Refactored. Thanks

Reply:
I think there is still a bug somewhere, link from footer point to product-page. 
Also, i think you might need to add a param to url -> products/beethoven, products/bartok, and then if you have a specific type, products/bartok/bm1

Reply:
Think we will do it when making series page